src/apps/voting/App.js

```diff
@@ -43,6 +46,10 @@ const App = React.memo(function App() {
     selectedVote,
     votes,
   } = useAppLogic()
+  const { representativeManager } = useAppState()
+  const showDelegateButton =
```

Comment:
maybe let's add to the showDelegateButton the other two conditions `account && !voterStatus.loading`